flink-cyber/metron-parser-chain/parser-chains-config-service/frontend/parser-chains-client/src/app/chain-page/components/custom-form/components/multi-input/multi-input.component.html

```diff
@@ -23,6 +23,7 @@
                         nz-input
                         [formControl]="controls[i]"
                         (keyup)="onChange(config)"
+                        (keyup.enter)="onAddClick()"
```

Comment:
We need a different action based on whether multipleValues = true or false.   In the current implementation, pressing enter on a field where multipleValues=false, adds another value to that field which won't work.